docs/source/examples/workspaces/random/README.rst

```diff
@@ -0,0 +1,4 @@
+Random
+======
+
+This workspace contains PinFiles, topologies, and layous that don't fit into another workspace, or that may be used by multiple workspaces
```

Comment:
Is this needed if there aren't any other files in the random workspace?

Reply:
There's a few files in there, but maybe #644 moves them out.  PinFile.dyasny.yml, PinFile.libvirt-dummy.yml, and PinFile.openstack-aws.yml.  There's also a few topologies in random/topologies that aren't used in any specific PinFile, but I didn't know if they could be deleted

Reply:
Yeah, fair enough. This is just coordination between the two PRs. Thanks for doing this work.